java/samples/sample-code/src/main/java/com/microsoft/semantickernel/syntaxexamples/Example29_Tokenizer.java

```diff
@@ -0,0 +1,84 @@
+package com.microsoft.semantickernel.syntaxexamples;
+
+import com.microsoft.semantickernel.GPT3Tokenizer;
+public class Example29_Tokenizer {
+    /**
+     * This sample shows how to count tokens using GPT tokenizer. The number of tokens affects
+     * API calls cost and each model has a maximum amount of tokens it can process and generate.
+     * This example is specific to OpenAI models, which use the tokenization described here:
+     * <a href="https://platform.openai.com/tokenizer">OpenAI tokenizer</a>
+     * If you use Semantic Kernel with other models, the tokenization logic is most probably different,
+     * and you should not use the GPT tokenizer.
+     */
+    public static void main(String[] args) {
+        // Example 1
+        String sentence = "Some text on one line";
+        int tokenCount = GPT3Tokenizer.encode(sentence).size();
+
+        System.out.println("---");
+        System.out.println(sentence);
+        System.out.println("Tokens: " + tokenCount);
+        System.out.println("---\n\n");
+
+        // Example 2
+        sentence = "⭐⭐";
+        tokenCount = GPT3Tokenizer.encode(sentence).size();
+
+        System.out.println("The following example contains emojis which require several tokens.");
+        System.out.println("---");
+        System.out.println(sentence);
+        System.out.println("Tokens: " + tokenCount);
+        System.out.println("---\n\n");
```

Comment:
This could be extracted out to a utility method

Reply:
My $.02 is that examples are harder to follow when you have to take a detour to some utility method. 

I'll add, however, that this could be easier to read if it used printf.

Reply:
It could, but as printing the tokenCount is what matters the most in the example I would prefer to leave it explicitly here. Also, because it keeps parity with the .NET example: https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/KernelSyntaxExamples/Example29_Tokenizer.cs